gallery/gallery/lib/pages/about.dart

```diff
@@ -47,12 +53,13 @@ class _AboutDialog extends StatelessWidget {
           crossAxisAlignment: CrossAxisAlignment.start,
           mainAxisSize: MainAxisSize.min,
           children: [
-            Center(
-              child: Text(
-                '$name',
-                style: textTheme.display1.apply(color: colorScheme.onPrimary),
-              ),
-            ),
+            FutureBuilder(
+                future: getVersionNumber(),
+                builder: (context, snapshot) => Text(
+                      snapshot.hasData ? '$name ${snapshot.data}' : '$name',
+                      style: textTheme.display1
+                          .apply(color: colorScheme.onPrimary),
+                    )),
```

Comment:
This is indented somewhat weirdly, I would try adding a trailing comma here. And maybe not using the => syntax.

Reply:
Fixed with a comma